packages/scandipwa/src/component/WishlistItem/WishlistItem.component.js

```diff
@@ -227,8 +227,13 @@ export class WishlistItem extends PureComponent {
         const { label, value } = option;
 
         return (
-            <span block="WishlistItemOption" key={ `${ label }-${ value }` }>
-                { `${ value} x ${label }` }
+            <span block="WishlistItem" elem="Option" key={ `${ label }-${ value }` }>
+                <div block="WishlistItem" elem="GroupedProductTitle">
+                    <strong>{ `${label}:` }</strong>
+                </div>
+                <div block="WishlistItem" elem="GroupedProductValue">
+                    { `${value}` }
```

Comment:
There is no reason to wrap the text value to the template literals if you don't add any additional values to the string.
Please use this value as is.